receiver/filelogreceiver/README.md

```diff
@@ -98,6 +98,15 @@ The header lines are not emitted by the receiver.
 
 Many parsers operators can be configured to embed certain followup operations such as timestamp and severity parsing. For more information, see [complex parsers](../../pkg/stanza/docs/types/parsers.md#complex-parsers).
 
+### Log Rotation
+
+Log Rotation is supported for File Log Receiver, but the following settings need to be configured:
+
+- `poll_interval`: `10ms` (can adjust to fit your use case)
+- `start_at`: `beginning`
```

Comment:
Please correct me if I'm wrong, but since log rotation involves creating and reading multiple files, start_at needs to be set to "beginning" in order to properly read all logs from newly created files, otherwise it may end up missing some logs if it's set to "end".

Additionally, these settings needed to be [changed for the tests involving log rotation](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/filelogreceiver/filelog_test.go#LL290C1-L290C1). The tests will always fail if start_at = "end". As for the poll interval, it was set to "10ms" in the tests, so I assumed that it was necessary for it to be set to a lower value than the default.

Reply:
The `start_at` setting is only relevant the first time the collector finds files. After that, any new files found are read from the beginning. This could be clarified in the description of `start_at` but it does say "At startup".

As for the test cases, the failures you are seeing are due to the way the test is setup. Specifically, we start the receiver and immediately begin writing lines to log files, but the receiver takes some non-zero amount of time to find these files once started. The result is that some lines may have been written to the files before the receiver sees them for the first time. You can see that the tests pass if we insert a small delay between starting the receiver and writing to the files.

Reply:
Thank you for clarifying, I'll remove that setting from the log rotation section. I still think that poll interval would matter as rapphil stated [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/23609#discussion_r1237602388), but if you think otherwise I can remove that setting as well.